src-ghc/Pact/Server/ApiServer.hs

```diff
@@ -147,11 +148,12 @@ checkHistoryForResult rks = do
   liftIO $ writeHistory hChan $ QueryForResults (rks,m)
   liftIO $ readMVar m
 
-pollResultToReponse :: HM.HashMap RequestKey CommandResult -> PollResponses
-pollResultToReponse m = PollResponses $ HM.fromList $ map (second crToAr) $ HM.toList m
+pollResultToReponse :: HM.HashMap RequestKey (CommandResult [TxLog Value]) -> PollResponses
+pollResultToReponse m = PollResponses $ HM.fromList $ map (second fullToHashLogCr) $ HM.toList m
 
-crToAr :: CommandResult -> ApiResult
-crToAr CommandResult {..} = ApiResult (toJSON _crResult) _crTxId Nothing
+fullToHashLogCr :: CommandResult [TxLog Value] -> CommandResult Hash
```

Comment:
hmm ... we should consider making the hashes consistent with Chainweb. 

@larskuhtz @emilypi with this change Pact logs will be Blake2b256 hashed. Is there any reason why we need this to be consistent with CW? Pact is consistently Blake2b256 so I'm inclined to stick with it here.

@larskuhtz this is one reason why it would have been better to stick with Blake2b across kadena products (Scalable and Pact use only blake) as we have to deal with UX inconsistencies like this.

Reply:
I think, we could switch to Blake2b easily. SHA512t_256 has very good performance, but not sure how much that matters and how Blake2b compares.

Generally, I don't see an issue with using different hashes for different operations. These hashes are not related and serve different purposes. For instance, we don't restrict TLS in chainweb-node to only use SHA512, because we use it in Merkle trees.

I think, as long as we stick to well established standards, it's fine to just use what ever hash fits best the respective use case.

Reply:
@slpopejoy I just noticed that my comment above wasn't very clear. So in summary: personally, I think it's fine to use different algorithms in different places, but I also don't see a reason not to use blake2 in the merkle trees or elsewhere in cw.

Reply:
ok well we'll stick with Blake2b in pact for now

Reply:
<img width="895" alt="image" src="https://user-images.githubusercontent.com/1369810/58138452-0a309d80-7beb-11e9-9a78-8ddffd7e8fad.png">

Reply:
<img width="895" alt="image" src="https://user-images.githubusercontent.com/1369810/58138477-26ccd580-7beb-11e9-8ef1-5c5b639d886d.png">

Reply:
<img width="895" alt="image" src="https://user-images.githubusercontent.com/1369810/58138500-4106b380-7beb-11e9-9759-07ca20b79dff.png">

Reply:
So, performance of `merkle-log` is the best when used with `blake2b`. Should we switch to `blake2b` for the Merkle trees in Chainweb? Are there any drawbacks?